_specifications/specification-3-16.md

```diff
@@ -2779,6 +2779,36 @@ _Response_:
 * result: `any` \| `null`
 * error: code and message set in case an exception happens during the request.
 
+#### <a href="#client_executeCommand" name="client_executeCommand" class="anchor">Execute a client command(:leftwards_arrow_with_hook:)</a>
+
+The `client/executeCommand` request is similar to `workspace/executeCommand` but is sent from the server to the client, to trigger command execution on the client. 
+
+_Client Capability_:
+* property path (optional): `client.executeCommand`
```

Comment:
This is the first client property path that starts with `client.`. Why not `workspace.`?

Should we be able to distinguish between workspace and textDocument commands?

Reply:
I would just make the property mirror the name of the message. That seems to be the pattern so far. So, if we call the message 'client/executeCommand' than I think we should call the property 'client.executeCommand'. 

But I personally don't care all that much about what exactly we call either the message or the property. 

So I'm okay if you suggest some other names for both and I'll just go along with whatever other people seem to like best.

If you want to suggest different name (for message or property), it would be best to suggest the full names you'd prefer (`workspace.` is only the prefix and I think we can't use `workspace.executeCommand` because that's already used for the client->server message).

Reply:
The only other client/ requests seem to be client/registerCapability and client/unregisterCapability. But those don't have a client capability. I'm fine with `client.executeCommand` :)

Reply:
So does that mean we can resolve the conversation? It sounds like you are fine leaving it as is? But I'm not sure that's what you are really saying :-).

Reply:
Yes, I meant that you can resolve the convo. Sorry for the confusion!